contracts/scripts/staking/constants.js

```diff
@@ -0,0 +1,9 @@
+const compensationData = {
+  type: 1,
+  rate: 25,
+  duration: 31104000,
```

Comment:
This is 360 days. We should use 365 days = 31536000 sec so that it is consistent with the user staking which has duration of 30,90 or 365 days.

Reply:
Note that after updating this, we should re-generate the file merkleProofedAccountsToBeCompensated.json for the DApp to use.

Reply:
Thanks for catching this one! Interestingly on the dapp we read the durations from the contract now rather than from the API serving this csv data. That's why this probably hasn't been obvious when testing the dapp.